frontend/src/pages/CollaborationPage.js

```diff
@@ -77,43 +111,120 @@ const CollaborationPage = () => {
     } catch (err) {
       console.log(err)
     }
-    navigate(homeUrl)
+
+    // After room is deleted, setIsPartnerLeft as true and setIsPartnerOnline as
+    // false so that either PartnerLeftAlertDialog or PartnerOfflineAlertDialog
+    // will not be shown if partner happens to submit of skip review first
+    setIsPartnerLeft(true)
+    setIsPartnerOnline(false)
+
+    handleLeaveRoomConfirmationCloseDialog()
+    handleReviewPartnerOpenDialog()
+  }
+
+  const renderPartnerOfflineAlertDialog = () => {
+    return (
+      <AlertDialog
+        dialogOpen={!isPartnerOnline && !isPartnerLeft}
```

Comment:
@Silvernitro PartnerOfflineAlertDialog will only be shown when partner is not online (!isPartnerOnline) and has not left (!isPartnerLeft). This is to ensure that PartnerOfflineAlertDialog is not shown when partner has left (isPartnerLeft).

Reply:
ok! thanks for the comments in the code too 👍🏻